src/assets/styles/base/base.scss

```
@@ -1,5 +1,5 @@
 @import "~assets/styles/abstracts/mixins.scss";
 @import "~assets/styles/themes/default.scss";
-@import "~assets/styles/base/bc-sans.scss";
+@import "~assets/styles/base/BCSans.scss";
```

```suggestion
@import "@bcgov/bc-sans/css/BCSans.css";
```
This seems to compile (after `npm i --save @bcgov/bc-sans`) for me and the new BCSans font files end up in dist/ but I'm not able to get the app running to check if they actually get used.

---

Thanks, looks like I made a mistake on the import address. I was able to get it working now.

---

Nice, glad we can use the package after all.